indico/modules/rb_new/controllers/backend.py

```diff
@@ -142,6 +144,9 @@ def _process(self, args):
         for room_id in availability:
             data = availability[room_id]
             data['room'] = rooms_schema.dump(data['room'], many=False).data
+            data.update({'blockings': serialize_blockings(data['blockings'])})
```

Comment:
data['blockings'] = ...

same for the two lines below

or change this to a single update call where you pass everything - but in that case also include the `*_days_available` entries there